src/wp-includes/class-wp-block.php

```diff
@@ -446,6 +447,19 @@ public function render( $options = array() ) {
 			}
 		}
 
+		$computed_attributes = $this->process_block_bindings();
+
+		// If there are computed attributes, update the attributes with the
+		// computed ones.
+		if ( ! empty( $computed_attributes ) ) {
+			// Merge the computed attributes with the original attributes
+			$this->attributes = array_merge( $this->attributes, $computed_attributes );
+		}
+
+		foreach ( $computed_attributes as $attribute_name => $source_value ) {
```

Comment:
This loop is only necessary when `$block_content` is not empty. Otherwise, there is nothing to replace.

Reply:
Makes sense. Done in 313af354adc682d85582d893439477f810e72279